sunpy/map/mapbase.py

```diff
@@ -663,9 +721,16 @@ def rotation_matrix(self):
         """
         Matrix describing the rotation required to align solar North with
         the top of the image.
-        Meta keyword : ``PC1_1``, ``PC1_2``, ``PC2_1``, ``PC2_2``, ``CD1_1``, ``CD1_2``, ``CD2_1``, ``CD2_2``.
-                        returns np matrix with ``PC1_1``, ``PC1_2``, ``PC2_1``, ``PC2_2``
-                        else with ``CD1_1``, ``CD1_2``, ``CD2_1``, ``CD2_2``.
+         .. rubric:: Metadata keywords:
+
+        * ``PC1_1``
```

Comment:
These are generally written as `PCi_j` and `CDi_j` also these are an either or kinda thing, i.e.
```
* First Available
  1. `PCi_j`
  2. `CDi_j`
```